code/modules/organs/organ_alien.dm

```diff
@@ -336,3 +336,51 @@
 	name = "vox cortical stack"
 
 /obj/item/organ/stack/vox/stack
+
+
+//WRYN ORGAN
+
+/obj/item/organ/wryn/hivenode
+	name = "antennae"
+	parent_organ = "head"
+
+/obj/item/organ/wryn/hivenode
+	name = "antennae"
+	organ_tag = "antennae"
+	icon = 'icons/mob/human_races/r_wryn.dmi'
+	icon_state = "antennae"
+
+
+//NUCLEATION ORGAN
+
+/obj/item/organ/nucleation
+	name = "nucleation organ"
+	icon = 'icons/obj/surgery.dmi'
+	desc = "A crystalized human organ. /red It has a strangely iridescent glow."
+
+/obj/item/organ/nucleation/resonant_crystal
+	name = "resonant crystal"
+	icon_state = "resonant-crystal"
+	organ_tag = "resonant crystal"
+	parent_organ = "head"
+
+/obj/item/organ/nucleation/strange_crystal
+	name = "strange crystal"
+	icon_state = "stramge-crystal"
```

Comment:
Possible typo? Item name is 'strange crystal', the icon_state is 'stramge-crystal'.